cahee/css/styles_cahee_header.css

```diff
@@ -83,3 +91,104 @@
     margin-left: 1em;
   }
 }
+
+@media screen and (max-width: 767px) {
+  #nav-container {
+    width: 100%;
+    height: 100vh;
+    pointer-events: none;
+  }
+
+  .logo-main {
+    display: none;
+  }
+
+  .logo-mobile {
+    display: block;
+    background-color: white;
+    position: absolute;
+    top: -75px;
+    left: -75px;
+    padding: 150px 200px 100px;
+    border-radius: 50%;
+    box-shadow: 0px 0px 8px 10px #888888;
+  }
+
+  .bg {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    visibility: hidden;
+    opacity: 0;
+    transition: opacity 0.3s, visibility 0s;
+    background: rgba(255, 255, 255, 0.699);
+  }
+
+  #nav-container:focus-within .bg {
+    visibility: visible;
+    opacity: 0.6;
+  }
+
+  #nav-container * {
```

Comment:
Be careful with this kind of selectors, since it will trigger every single element inside `#nav-container`

Reply:
I'm getting the risks, in this specific case, it's exactly what I wanted since all elements were hidden in another screen size.